source/git-util.js

```diff
@@ -108,3 +108,9 @@ exports.verifyTagDoesNotExistOnRemote = async tagName => {
 exports.commitLogFromRevision = revision => execa.stdout('git', ['log', '--format=%s %h', `${revision}..HEAD`]);
 
 exports.push = () => execa('git', ['push', '--follow-tags']);
+
+exports.getLastCommit = () => execa.stdout('git', ['log', '--max-count=1', '--pretty=%B']);
+
+exports.deleteTag = tagName => execa('git', ['tag', '--delete', tagName]);
```

Comment:
I think these should be async non-inline functions instead. Right now, the `deleteTag` method actually returns an object of `stdout` and `stderr`. It should not return anything, so we don't accidentally depend on it.

```js
exports.deleteTag = async tagName => {
	await execa('git', ['tag', '--delete', tagName]);
};
```

Same with all the other ones that should not return anything.